scap-security-guide.spec

```diff
@@ -97,8 +97,8 @@ rm %{buildroot}/%{_docdir}/%{name}/Contributors.md
 
 %files
 %{_datadir}/xml/scap/ssg/content
-# No kickstarts for Fedora
-%if 0%{?_rhel_like}
+# No kickstarts for Fedora or CentOS Stream 10+
+%if 0%{?_rhel_like} && 0%{?_rhel_like} < 10
```

Comment:
Should that be <= 10? This would only match 11+.

Reply:
No, the condition is inverted.

* ie. for RHEL-9, it will be `%if 09 && 09 < 10` ==> true, so the the kickstarts directory is included
* ie. for Fedora, it is `%if 0 && 0 < 10` ==> false, so it is **not** included (we don't build kickstarts for Fedora)
* ie. for RHEL-10, it is `%if 010 && 10 < 10` ==> false, so it is **not** included
* ie. for RHEL-11, it is `%if 011 && 11 < 10` ==> false as well

The `true` match is whether `kickstarts/` should be there.

Reply:
Thanks. After thought about it again, I realized that I inverted the logic in my mind.